README.md

```diff
@@ -88,7 +88,7 @@ String id = jwt.getId();
 
 #### Time Validation
 
-The JWT token may include DateNumber fields that can be used to validate that the token was issued in a past date `"iat" < TODAY` and that the expiration date is in the future `"exp" > TODAY`. This library includes a method that checks both of this fields and returns the validity of the token. If any of the fields is missing they wont be considered. You must provide a positive amount of seconds as leeway to consider in the Date comparison.
+The JWT token may include DateNumber fields that can be used to validate whether the token was issued in a past date (`"iat" < TODAY`) or whether the expiration date is in the future (`"exp" > TODAY`). This library includes a method that checks both of these fields and returns the validity of the token. If any of the fields are missing they won't be considered. You must provide a positive amount of seconds as leeway, that will be considered in the Date comparison.
```

Comment:
I think this section still needs work. 

- I don't think the "whether" substitution is much better, maybe"if" instead?
- The last sentence could be more clear ... also "a positive amount of seconds as leeway" is awkward